src/python/pants/backend/helm/util_rules/renderer.py

```diff
@@ -271,7 +271,10 @@ async def setup_render_helm_deployment_process(
 
     merged_digests = await Get(Digest, MergeDigests(input_digests))
 
-    release_name = request.field_set.release_name.value or request.field_set.address.target_name
+    release_name = (
+        request.field_set.release_name.value
+        or request.field_set.address.target_name.replace("_", "-")
```

Comment:
Is this a Helm convention to use dashes not underscores?

Reply:
yes, and it's also forbidden to use underscores to name a release, so the easy way out was transforming them into dashes...